src/cf-forms/usage.md

```diff
@@ -130,26 +131,54 @@ Inputs should always be paired with a `label` for accessibility reasons.
 
 ### Basic text inputs
 
-<label class="a-label" for="text-input-example">A text input</label>
+<label class="a-label" for="textinput-example">A text input</label>
```

Comment:
Okay, I think `textarea` is that way because it follows the format of the <textarea> html, which if this followed the html would be two words, but not a biggie.